pkg/networkmanager/team.jsx

```diff
@@ -151,7 +151,8 @@ export const TeamDialog = ({ connection, dev, settings }) => {
         <NetworkModal dialogError={dialogError}
                       idPrefix={idPrefix}
                       onSubmit={onSubmit}
-                      title={_("Team settings")}
+                      title={!connection ? _("Add team") : _("Edit team settings")}
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-19209-20230829-062057-dc4a837a-fedora-38-devel//Coverage/lcov/github-pr.diff.gcov.html)